docs/campaign.rst

```diff
@@ -227,6 +227,15 @@ srun_options
 When the `srun` execution layer is enabled, a list of providing additional
```

Comment:
Should maybe explain the different behavior when constraint arguments are given

Reply:
Good idea. I will update the documentation accordingly.